README.md

```diff
@@ -2,49 +2,40 @@
 
 **This is a Real Time Chat Application**
 
--   Preview - [WaveChat (Coming Soon)](#)
+- Preview - [WaveChat (Coming Soon)](#)
 
 This is a real time chat app made for college/university students. In the recent couple years, the covid-19 pandemic
 has changed how students of all level interact with each other. It has been really tough for students
 to meet new people and make friends over the internet. This app was made for students to find other students
 of their similar level and make new friends. It also makes it easy for students of a certain course or section to
 communicate with each other about their activities and help each other.
 
-# Stack
+## Stack
 
--   [Typescript](https://www.typescriptlang.org/)
--   [Next.js](https://nextjs.org/)
--   [React](https://reactjs.org)
--   [Apollo](https://www.apollographql.com/)
--   [Zustand](https://github.com/pmndrs/zustand)
--   [TailwindCSS](https://tailwindcss.com/)
--   [Nodejs](https://nodejs.org/en/)
--   [GraphQL](https://graphql.org/)
--   [GiraphQL - SchemaBuilder](https://giraphql.com/)
--   [Socket.IO](https://socket.io/)
--   [Prisma](https://www.prisma.io/)
--   [PostgresSQL](https://www.postgresql.org/)
--   [Docker](https://www.docker.com/)
+| [![TypeScript](images/typescript.png)](https://www.typescriptlang.org/)| [![TypeScript](images/nextjs.png)](https://nextjs.org/)| [![TypeScript](images/react.png)](https://reactjs.org) | [![TypeScript](images/apollo.png)](https://www.apollographql.com/) |
+|---|---|---|---|
+| [![TypeScript](images/tailwind.png)](https://tailwindcss.com/) | [![TypeScript](images/nodejs.png)](https://nodejs.org/en/)| [![TypeScript](images/graphql.png)](https://graphql.org/) | [![TypeScript](images/socket.png)](https://socket.io) |
```

Comment:
you can use `<img>` tag with html attributes for styling

Reply:
we can only control width and height there and that wont be really helpful in this case, tbh.